tools/ctl/rundex/rundex.go

```diff
@@ -132,8 +132,11 @@ func DoQuery[T any](ctx context.Context, q firestore.Query, fn func(*firestore.D
 }
 
 func cleanVerdict(m string) string {
+	m = strings.Replace(m, "getting strategy: fetching inference: failed to infer strategy:", "inference failure:", -1)
```

Comment:
can we add a short note that we expect this to be the inference error prefix for rebuilds so we strip it to ensure inference errors are properly sanitized?